test/filters/not_test.rb

```diff
@@ -20,13 +20,13 @@ def test_to_hash_return_proc
   def test_to_hash_change_last_term_filter_value
     filter_class = MiniTest::Mock.new
     filter_class.expect :valid?, true
-    filter_class.expect :to_hash, {query: {filtered: {filter: {and: {term: {a: 1, b: 2}}}}}}
+    filter_class.expect :to_hash, {query: {filtered: {filter: {and: [{term: {a: 1, b: 2}}]}}}}
```

Comment:
Line is too long. [94/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.